src/app/static/src/app/components/projects/ProjectHistory.vue

```diff
@@ -26,7 +26,7 @@
                     <a @click="deleteProject($event, p.id)" href="" v-translate="'delete'"></a>
                 </div>
                 <div class="col-md-2 project-cell">
-                    <a @click="copyProject($event, p.id)" href="" v-translate="'copyToNewProject'"></a>
+                    <a @click="copyVersion($event, p.id, p.versions[0].id)" href="" v-translate="'copyLatestToNewProject'"></a>
```

Comment:
Once all of the tickets affecting the projectHistory have been combined (version name, rename project, and this one), we could replace all the links with icons that have tooltips.